dd-sdk-android/src/test/kotlin/com/datadog/android/utils/BatchExt.kt

```diff
@@ -0,0 +1,10 @@
+package com.datadog.android.utils
+
+import com.datadog.android.core.internal.domain.Batch
+import com.google.gson.JsonArray
+import com.google.gson.JsonParser
+
+internal val Batch.logs: JsonArray
```

Comment:
maybe rename this extension as `Batch.asJsonArray` instead. We might use it for more than logs ;)